### ISSUE TITLE
Errors when previewing a certificate template

### ISSUE THREAD

**Issue by jeherve** [woocommerce/sensei-certificates#98]
I get the following errors hitting the "Preview" button when editing a certificate template.

```
Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; tFPDF has a deprecated constructor in /wp-content/plugins/sensei-certificates/lib/tfpdf/tfpdf.php on line 13

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; TTFontFile has a deprecated constructor in /wp-content/plugins/sensei-certificates/lib/tfpdf/font/unifont/ttfonts.php on line 38
FPDF error: Some data has already been output, can't send PDF file (output started at /wp-content/plugins/sensei-certificates/classes/class-woothemes-sensei-certificate-templates.php:368)
```

Using PHP 7 on that test site.

**Comment by aheckler** [comment]
536589-zd-woothemes

**Comment by pgk** [comment]
possibly addressed in #120 but upgrading tFPDF is a good idea

**Comment by andrewjohnstrong** [comment]
Why has this been closed? The bug hasn't been fixed.

**Comment by danjjohnson** [comment]
It has been closed in favour of the two other issues referenced above.

**Comment by danjjohnson** [comment]
Confirmed this is fixed in 1.0.14
